index.html

```diff
@@ -3111,26 +3111,63 @@ <h2>
           <li>Set <var>response</var>.<a>[[\retryPromise]]</a> to
           <var>retryPromise</var>.
           </li>
-          <li data-link-for="PaymentValidationErrors" data-tests=
-          "PaymentValidationErrors/retry-shows-error-member-manual.https.html">
-          If <var>errorFields</var>'s <a>paymentMethod</a> member was passed,
-          and if required by the specification that defines
-          <var>response</var>'s <a>payment method</a>, then <a data-cite=
-          "WEBIDL#dfn-convert-ecmascript-to-idl-value">convert</a>
-          <var>errorFields</var> <a>paymentMethod</a> to an IDL value of the
-          type specified there. Otherwise, <a data-cite=
-          "WEBIDL#dfn-convert-ecmascript-to-idl-value">convert</a> to
-          <a data-cite="WEBIDL#idl-object">object</a>.
-          </li>
-          <li>If conversion results in a <a data-cite=
-          "WEBIDL#dfn-exception">exception</a> <var>error</var>:
+          <li>If <var>errorFields</var> was passed:
             <ol>
-              <li>Reject <var>retryPromise</var> with <var>error</var>.
+              <li>Optionally, show a warning in the developer console if any of
```

Comment:
FYI soon you and other specs will be able to link to a formal definition here, per https://github.com/whatwg/console/pull/151. No action for now though.

Reply:
Good to know! thanks!